internal/cli/atlas/clusters/region_tier_autocomplete.go

```diff
@@ -38,7 +39,11 @@ type autoCompleteOpts struct {
 
 func (opts *autoCompleteOpts) autocompleteTier() cli.AutoFunc {
 	return func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
-		opts.parseFlags(cmd)
+		if err := opts.parseFlags(cmd); err != nil {
+			cobra.CompErrorln(fmt.Sprintf("failed to parse flags: %v", err))
```

Comment:
FYI this messages only show when debugging autocompletion so they are not usually available to users